akka-docs/src/main/paradox/typed/cluster.md

```diff
@@ -95,15 +112,41 @@ Scala
 Java
 :  @@snip [BasicClusterExampleTest.java](/akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/BasicClusterExampleTest.java) { #cluster-leave-example }
 
-## Serialization
-
-See [serialization](https://doc.akka.io/docs/akka/current/scala/serialization.html) for how messages are sent between
-ActorSystems. Actor references are typically included in the messages,
-since there is no `sender`. To serialize actor references to/from string representation you will use the `ActorRefResolver`.
-For example here's how a serializer could look for the `Ping` and `Pong` messages above:
-
-Scala
-:  @@snip [PingSerializer.scala](/akka-cluster-typed/src/test/scala/docs/akka/cluster/typed/PingSerializer.scala) { #serializer }
-
-Java
-:  @@snip [PingSerializerExampleTest.java](/akka-cluster-typed/src/test/java/jdocs/akka/cluster/typed/PingSerializerExampleTest.java) { #serializer }
+ 
```

Comment:
Missing a few sections here that exists in the Classic page
- Joining to Seed Nodes
- Downing
- Leaving

Maybe some parts of the following sections from classic page are also relevant here:
- How To Startup when Cluster Size Reached
- How To Startup when Member is Up
- How To Cleanup when Member is Removed